src/olympia/addons/models.py

```diff
@@ -2099,3 +2105,14 @@ class ReusedGUID(ModelBase):
     guid = models.CharField(max_length=255, null=False)
     addon = models.OneToOneField(
         Addon, null=False, on_delete=models.CASCADE, unique=True)
+
+
+class GitExtraction(ModelBase):
```

Comment:
mmm maybe this could be renamed to `AddonGitExtraction` to avoid potential conflicts in the future but the relation name would become `addongitextraction`...

Reply:
You can customize the related name... But I don't think it matters at the moment. This is fine like this.